studio/components/layouts/AccountLayout/AccountLayout.tsx

```diff
@@ -78,6 +78,13 @@ const AccountLayout = ({ children, title, breadcrumbs }: PropsWithChildren<Accou
                 href: `/account/me`,
                 key: `/account/me`,
               },
+              {
```

Comment:
This will be behind feature flag.

Reply:
Sounds correct to me!
Small thing, but would you mind also adding this 'Security' link to the V2 navigation?

Reply:
^ yeah this one's okay - IS_PLATFORM is just what we use to discern between self-hosted and hosted (its not a feature flag that we use through configcat)

should be present despite having mfa set up or not